styles/style.css

```diff
@@ -0,0 +1,135 @@
+/* Custome Properties */
+:root {
+  /* colors */
+  --primary-clr: #1a2236;
+  --secondary-clr: #fff;
+
+  /* fonts */
+  --ff-headings: 'Playfair Display', serif;
+  --ff-normal: 'Poppins', sans-serif;
+
+  /* font-size */
+  --fs-50: 3.125rem;
+  --fs-16: 1rem;
+  --fs-14: 0.875rem;
+
+  /* font-weights */
+  --fs-500: 500;
+  --fs-400: 400;
+}
+
+/* Resets */
+* {
+  box-sizing: border-box;
+  font: inherit;
+  padding: 0;
+  margin: 0;
+}
+
+html,
+body {
+  height: 100%;
+  text-rendering: optimizeSpeed;
+}
+
+img {
+  display: block;
+  width: 100%;
+}
+
+/* Typography */
+body {
+  color: var(--primary-clr);
+  font-family: var(--ff-normal);
+  font-size: var(--fs-14);
+  font-weight: var(--fw-400);
+}
+
+h1 {
+  font-family: var(--ff-headings);
+  font-size: var(--fs-50);
+}
+
+button {
+  font-size: var(--fs-16);
+  font-weight: var(--fw-500);
+}
+
+/* Mobiles Styles */
+.bg-decoration {
+  background: url("../assets/images/image_geometry_1.svg") no-repeat;
+}
+
+.section {
+  display: flex;
+}
+
+.container {
+  padding-inline: 1em;
+}
+
+/* Header Section */
+.header-section {
+  justify-content: flex-end;
+}
+
+.menu {
+  background: none;
+  border: none;
+  height: fit-content;
+  margin-top: 1.1875em;
+}
+
+/* Hero Section */
+.hero-section {
+  flex-direction: column;
+}
+
+.hero-content {
+  margin: 4.5em 0 4em 0;
+}
+
+.hero-content h1 {
+  max-width: 183px;
+  line-height: 50px;
+}
+
+.hero-content p {
+  margin: 1.25em 0 2.5em 0;
+  width: max(445px, 100%);
+  line-height: 21px;
+}
```

Comment:
- [ ] Kindly adjust your width on the introduction paragraph because it have take more than 100% width of the screen :wink:

  - ### Your design
 
   ![image](https://user-images.githubusercontent.com/87197412/185754760-f5a77ac3-1594-4662-82f2-be5118db7abf.png)


  - ### Figma design
  
   ![image](https://user-images.githubusercontent.com/87197412/185754775-e83ed99b-abbd-4af0-9bef-fe5d0af41ad3.png)

Reply:
Hi @levy002 , got it.